.github/workflows/build.yaml

```
@@ -39,6 +57,7 @@ jobs:
 
     strategy:
       matrix:
+        pipeline: [ "${{ github.event.inputs.pipeline_name }}" ]
         service: [ "${{ github.event.inputs.service }}" ]
```

actionlint (1.7.8)

property "service" is not defined in object type {pipeline_name: string}

(expression)